db/migrate/20210517215935_remove_appointment_from_communication_and_add_patient.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveAppointmentFromCommunicationAndAddPatient < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :communications, :appointment_id
+    add_reference :communications, :patient, null: true, foreign_key: true, type: :uuid
```

Comment:
Before we drop `appointment_id`, should we add a data migration to copy over `patient_id` for existing `communications`?

Reply:
We briefly considered that but didn't pursue it. Maybe we should, then we wouldn't have to make patient optional for the notification model.

Reply:
I've now added a backfill.